tools/online_editor/src/preview_widget.ts

```diff
@@ -9,14 +9,7 @@ import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
 
 import slint_init, * as slint from "@preview/slint_wasm_interpreter.js";
 
-let was_initialized = false;
-
-async function setup_preview() {
-  if (!was_initialized) {
-    await slint_init();
-  }
-  was_initialized = true;
-}
+const ensure_slint_wasm_bindgen_glue_initialized = slint_init();
```

Comment:
it is not obvious to me when reading the code that this is a future. I guess a comment of what this line is for would help.

Reply:
I can add an extra type annotation tomorrow:)